integrations/wootric/lib/index.js

```diff
@@ -101,14 +98,21 @@ Wootric.prototype.track = function(track) {
  * @param {Page} page
  */
 
-Wootric.prototype.page = function() {
-  // Only track page if we haven't already tracked it
-  if (this.lastPageTracked === window.location) {
-    return;
-  }
```

Comment:
I'm new to this repo, but why are you deleting this? Is this not need anymore?

Reply:
We are removing that to allow multiple calls to `page()`. We will be triggering surveys based on the page name that you can pass as an argument, for example`page("billing"). The page name variable is not related to the URL so returning early because the page was already tracked is not an option since it could lead to ignoring valid calls to `page()`

Reply:
Cool, that makes sense.